### ISSUE TITLE
only german VISUM files are supported (trac #1767)

### ISSUE THREAD

**Issue by behrisch** [eclipse/sumo#1767]
and no error message is given to the user which would indicate an unsupported language

Migrated from http://sumo.dlr.de/ticket/1767
```json
{
    "status": "new", 
    "changetime": "2015-09-21T12:14:05Z", 
    "description": "and no error message is given to the user which would indicate an unsupported language", 
    "reporter": "namdre", 
    "cc": "", 
    "resolution": "", 
    "_ts": "1442837645966007", 
    "component": "net building (NETGEN, NETCONVERT)", 
    "summary": "only german VISUM files are supported", 
    "priority": "major", 
    "keywords": "", 
    "time": "2015-05-04T10:37:19Z", 
    "milestone": "1.0.0", 
    "owner": "", 
    "type": "defect"
}
```

**Comment by namdre** [comment]
the groundwork is laid. Now only two things remain:
* add more keywords to the enum / StringBijection
* complete mapping files / add languages

**Comment by namdre** [comment]
fixed via #6339
